index.bs

```diff
@@ -28,11 +28,6 @@ Ignored Vars: e
 </pre>
 
 <pre class="anchors">
-urlPrefix: https://html.spec.whatwg.org/multipage/; spec: HTML
-    text: StructuredClone; urlPrefix: infrastructure.html; type: abstract-op
-    text: Transfer; url: infrastructure.html#transfer-abstract-op; type: abstract-op
-    text: WebSocket; url: comms.html#websocket; type: interface
-    text: transferable objects; url: infrastructure.html; type: dfn
```

Comment:
Recent changes to the HTML spec to export these terms automatically instead of requiring manual import.